lib/new-flatfile.ts

```diff
@@ -3,11 +3,34 @@ import { ListWorkbooksRequest } from "@flatfile/api/api";
 import { getSpaceConfig } from "./flatfile";
 
 // TODO: Need to take in per-workflow API key here
-const flatfileClient = () => {
-  const token = process.env.FLATFILE_API_KEY;
+
+export enum FlowTypes {
+  Onboarding = "onboarding",
+  Embedded = "embedded",
+  FileFeed = "filefeed",
+  Dynamic = "dynamic",
+}
+
+const flatfileClient = (flowName: string) => {
```

Comment:
type this as `FlowTypes` (or maybe `FlowType` singular on this type?) so that typescript enforces this